pymeasure/instruments/srs/sr830.py

```diff
@@ -26,7 +26,8 @@
 import time
 import numpy as np
 from enum import IntFlag
-from pymeasure.instruments import Instrument, discreteTruncate
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import discreteTruncate
```

Comment:
The `discreteTruncate` import should probably be added to the next line, instead.

Reply:
Added to the other validator imports